automation/test.sh

```diff
@@ -466,7 +468,7 @@ add_to_label_filter '(!exclude-native-ssh)' '&&'
 # Single-node single-replica test lanes obviously can't run live migrations,
 # but also currently lack the requirements for SRIOV, GPU, Macvtap and MDEVs.
 if [[ $KUBEVIRT_NUM_NODES = "1" && $KUBEVIRT_INFRA_REPLICAS = "1" ]]; then
-  add_to_label_filter '(!(SRIOV,GPU,Macvtap,VGPU,sig-compute-migrations,requires-two-schedulable-nodes))' '&&'
+  add_to_label_filter '(!(SRIOV,GPU,Macvtap,VGPU,sig-compute-migrations,requires-two-schedulable-nodes,persistent-ips))' '&&'
```

Comment:
since persistent-ips aren't sig-network, maybe we don't need this here anymore
worth to check please
will be cleaner

Reply:
Ahh wait this is not related to sig-network, since or persistent-ips are running live migration we need a pair of nodes, so it make sense to discard it if there is just one node.

Reply:
true but we don't set KUBEVIRT_NUM_NODES / KUBEVIRT_INFRA_REPLICAS where TARGET=kind-ovn
just on some special lanes as far as i remember
anyhow whatever you prefer